### DISCUSSION TITLE
Duplicate Pull Requests Bug

### DISCUSSION THREAD

**Discussion by NicholasLYang** [community/community#15803] [Other Feature Feedback, Questions, & Ideas]
Hi,

In the process of debugging a workflow, I opened two PRs from the same branch, one from [a fork](https://github.com/rome/tools/pull/2521), one from the [main repo](https://github.com/rome/tools/pull/2522). I noticed that both reported that the deploy_playground workflow failed. However, when I navigated to the workflow results, I realized that actually the workflow had succeeded [on the main repo](https://github.com/rome/tools/actions/runs/2246472661) and failed [on the fork](https://github.com/rome/tools/actions/runs/2246474399). Therefore it seems like GitHub is getting confused about which workflow is linked to which PR.

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
